understanding/accessible-authentication-minimum.html

```diff
@@ -125,203 +122,122 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
                   
-                  <p>If a <a href="https://www.w3.org/TR/turingtest/">CAPTCHA</a> is used as part of an authentication process, there must be a method that does not
-                     include a cognitive function test, unless it meets the exception. If the test is based
-                     on something the website has set such as remembering or transcribing a word, or recognizing
-                     a picture the website provided, that would be a cognitive functional test. Recognizing
-                     objects, or a picture the user has provided is a cognitive function test; however,
-                     it is excepted at the AA level.
+                  <p><a href="https://www.w3.org/TR/turingtest/">CAPTCHA</a> が認証プロセスの一部として使用される場合、例外を満たさない限り、認知機能テストを含まない方法が存在しなければならない。単語を記憶したり転記する、ウェブサイトが提供する画像の認識など、ウェブサイトが設定したものに基づいているテストである場合、それは認知機能テストになる。物体、又は利用者が提供した画像の認識も認知機能テストではあるが、AA レベルでは例外となる。
                   </p>
                   
                   
-                  <p>An object in this context means the general English definition ("a material thing
-                     that can be seen and touched") and can include vehicles and animals. If the test goes
-                     beyond recognition (e.g. multiply the number cats by the number of dogs), that does
-                     not meet the exception.
+                  <p>この文脈における物体とは、一般的な英語の定義 (「見て触れられる物質的なもの」) を意味し、乗り物及び動物を含むことがある。テストが認識を超える場合 (例: 猫の数に犬の数を掛ける)、例外を満たさない。
                   </p>
                   
                   
-                  <p>Some forms of object recognition may require an understanding of a particular culture.
-                     For example, taxis can appear differently in different locales. This is an issue for
-                     many people, including people with disabilities, but it is not considered an accessibility-specific
-                     issue.
+                  <p>物体の認識の一部の形式では、特定の文化の理解を必要とする場合がある。例えば、タクシーは地域によって外観が異なる場合がある。これは障害のある人を含む多くの人々にとっての問題であるが、アクセシビリティ特有の問題とはみなされていない。
                   </p>
                   
                   
-                  <p>Some CAPTCHAs and cognitive function tests used for authentication may only appear
-                     in certain situations, such as when ad blockers are present, or after repeated incorrect
-                     password entry. This criterion applies when these tests are used regardless of whether
-                     they are used every time or only triggered by specific scenarios.
+                  <p>認証に使用される一部の CAPTCHA 及び認知機能テストは、広告ブロッカーが存在する場合、間違ったパスワードを繰り返し入力した場合など、特定の状況でしか表示されないことがある。この基準は、これらのテストが毎回使用されるか、特定のシナリオによってのみトリガーされるかに関係なく、これらのテストが使用されるときに適用される。
                   </p>
                   
                   
-                  <p>There are a number of technologies that can be employed to prevent scripted abuse
-                     of the authentication process.
+                  <p>認証プロセスのスクリプトによる悪用を防ぐために使用できる技術は多数存在する。
```

Comment:
ここの scripted abuse は「スクリプトで自動化された濫用」というような意味で、主に bot による大量アクセスを指します。今の訳だと不正アクセスっぽいニュアンスにも読めてわかりにくいかもしれません。
しかし妙案はなく……
「スクリプトによる認証プロセスの濫用」とか?

```suggestion
                  <p>スクリプトによる認証プロセスの濫用を防ぐために使用できる技術は多数存在する。
```